lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -60,6 +61,24 @@ function dosomething_northstar_verify_user($credentials) {
     'data' => json_encode($credentials),
   ]);
 
+  // Add to request log
+  if(variable_get('dosomething_northstar_log')) {
+    // Don't log plaintext password.
+    $credentials['password'] = '*****';
```

Comment:
Yeah, we could do that. On the other hand, I wonder if it's worth doing an `isset` check and only masking the password like this if the field is in the payload... could catch if we're somehow not sending it on every request.